third_party/freedom-typings/freedom-common.d.ts

```diff
@@ -102,7 +113,8 @@ declare module freedom {
     // We use this specification so that you can reference freedom sub-modules by
     // an array-lookup of it's name. One day, maybe we'll have a nicer way to do
     // this.
-    [moduleName:string] : Function;
+    // TODO: explore how to use FreedomModuleFactoryManager.
+    [moduleName:string] : any;
```

Comment:
Not sure from the comment thread precisely what the question is, but if it's about directly accessing freedom modules dependencies from the core environment, then actually I think that's not possible - they can only be accessed through whatever API is specified in the manifest. If it's about something else then I guess I could use a short description of a hypothetical use case.